css/signin.css

```diff
@@ -1,40 +1,139 @@
 /* Base body styling for consistent layout */
+* {
+    margin: 0;
```

Comment:
nice! doing this allows us to not worry about default styling that browsers put on to some tags! great shout